plugins/UM3NetworkPrinting/src/ClusterUM3OutputDevice.py

```diff
@@ -574,6 +580,16 @@ def _updatePrintJob(self, print_job: PrintJobOutputModel, data: Dict[str, Any])
         if not status_set_by_impediment:
             print_job.updateState(data["status"])
 
+        print_job.updateConfigurationChanges(self._createConfigurationChanges(data["configuration_changes_required"]))
+
+    def _createConfigurationChanges(self, data: List[Dict[str, Any]]) -> List[ConfigurationChangeModel]:
+        result = []
+        for change in data:
+            result.append(ConfigurationChangeModel(type_of_change=change["type_of_change"],
```

Comment:
Why re-create them every update? I'm a bit afraid that this wil wreak havoc on the garbage collect for larger clusters.

Reply:
Re-creating on every update is much simpler less buggy than some kind of funky in-place update. We are not talking about many objects and you have to keep in mind that when doing anything in Python it creates and destroys objects like a madman. I strongly suspect that recreating is just faster than doing it in place.

Reply:
For example, that `for ... in` loop creates an iterator object and terminates when the collection creates an exception object and throws it. That is just one simple loop.

Reply:
Well, sure, but this is also creating a bunch of QObjects and matching properties. So it's doing a tad bit more. Especially since the update isn't checking if a signal needs to be emitted (so it's constantly triggering the re-drawing)